packages/components/src/__actions__/Menu/v3/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from "./Menu"
+export * from "./MenuItem"
+export * from "./MenuTrigger"
+// export * from "./SubmenuTrigger"
```

Comment:
Given its not accessible I think we should just remove the `SubmenuTrigger` and all related until we decide its a feature to have. I'm concerned we'll forget and let someone enable it by accident.

Reply:
Fair, I'll remove it all in a single commit once we're pretty settled on this PR going in, so that it's easy to see the code if we want to reference it later